bootstrap/spigot/src/main/java/org/geysermc/platform/spigot/world/GeyserSpigot1_11CraftingListener.java

```diff
@@ -94,7 +95,7 @@ public void onPlayerJoin(PlayerJoinEvent event) {
     }
 
     public void sendServerRecipes(GeyserSession session) {
-        int netId = RecipeRegistry.LAST_RECIPE_NET_ID;
+        AtomicInteger netId = InventoryUtils.LAST_RECIPE_NET_ID;
```

Comment:
Wouldn't this change mean that the `LAST_RECIPE_NET_ID` is incremented for all instances? That's not the intention of this field - it's supposed to be updated per-session (which may have different recipes), whereas the last recipe ID represents the shared, built-in recipe IDs.